code/modules/clothing/suits/armor.dm

```diff
@@ -229,7 +229,7 @@
 			if(M == owner)
 				continue
 			owner.Beam(M,icon_state="lightning[rand(1, 12)]",icon='icons/effects/effects.dmi',time=5)
-			M.adjustFireLoss(25)
+			M.adjustFireLoss(10)
```

Comment:
I could try to make it electrocute act without stunning but I'm not sure if that's possible. If I did, it would take into account the target's defences.